examples/for-tests/src/App.js

```diff
@@ -403,14 +409,14 @@ function getThirdPartyEmailPasswordConfigs() {
         onHandleEvent: async (context) => {
             console.log(`ST_LOGS THIRD_PARTY_EMAIL_PASSWORD ON_HANDLE_EVENT ${context.action}`);
         },
-
         useShadowDom,
         palette: theme.colors,
         emailVerificationFeature: {
             mode: emailVerificationMode,
         },
         resetPasswordUsingTokenFeature: {},
         signInAndUpFeature: {
+            disableDefaultImplementation,
```

Comment:
If this is set to `false`, you need to render the feature component manually in a route - I don't see that change in the for-tests application.. how are the tests passing?

Reply:
You're right. The behaviour I'm testing is:
- if you set `disableDefaultImplementation: true` on a feature
- then supertokens feature UI shouldn't render on `/default-path-for-feature` route

There are two different behaviours that this usecase is composed of:
1. Not rendering supertokens UI on `/somepath` - this is tested, as this is the primary concern for using this feature by users
2. Rendering some feature component outside - this is not tested and I would leave that for a unit test, since the behaviour is no different from rendering it normally (without `disableDefaultImplementation`). Unless it interacts with the previous behaviour in some special way that I didn't discover.

Let me know please for the second case if there's something I missed.

Reply:
- I don't think I understand point number 1. What is `/somepath` supposed to show?
- So are you simply testing that the feature does not show when you set `disableDefaultImplementation: true`?

Reply:
> - I don't think I understand point number 1. What is /somepath supposed to show?

It's just for illustrative purpose - the real paths we don't render on are `/auth`, `/auth/verify-email`, `/auth/reset-password`. I could have been clearer, sorry.

> - So are you simply testing that the feature does not show when you set disableDefaultImplementation: true?

Exactly. To be specific, I'm testing that it shows with `disableDefaultImplementation: false` and then that it doesn't with `disableDefaultImplementation: true`.

Reply:
I'm going to resolve this, based on our discussion on the call.